Add new properties that were missing (#83896)
Azure/azure-content: articles/logic-apps/quickstart-create-first-logic-app-workflow.md

   | **Resource Group** | <*Azure-resource-group-name*> | The [Azure resource group](../azure-resource-manager/management/overview.md#terminology) name, which must be unique across regions. This example uses "My-First-LA-RG". |
   | **Type** | **Consumption** | The logic app resource type and billing model to use for your resource: <p><p>- **Consumption**: This logic app resource type runs in global, multi-tenant Azure Logic Apps and uses the [Consumption billing model](logic-apps-pricing.md#consumption-pricing). This example uses this **Consumption** model. <p>- **Standard**: This logic app resource type runs in single-tenant Azure Logic Apps and uses the [Standard billing model](logic-apps-pricing.md#standard-pricing). |
   | **Logic App name** | <*logic-app-name*> | Your logic app resource name, which must be unique across regions. This example uses "My-First-Logic-App". <p><p>**Important**: This name can contain only letters, numbers, hyphens (`-`), underscores (`_`), parentheses (`(`, `)`), and periods (`.`).  |

>    | **Publish** | **Workflow** | Only available for Standard Type. <p><p>The deployment destination for your logic app. By default, **Workflow** is selected for deployment to single-tenant Azure Logic Apps. Azure creates an empty logic app resource where you have to add your first workflow. <p><p>**Note**: Currently, the **Docker Container** option requires a [*custom location*](../azure-arc/kubernetes/conceptual-custom-locations.md) on an Azure Arc enabled Kubernetes cluster, which you can use with [Azure Arc enabled Logic Apps (Standard)](azure-arc-enabled-logic-apps-overview.md). The resource locations for your logic app, custom location, and cluster must all be the same. |

Reviewer: @sandroasp Thanks for contribution! We didn't include the **Publish** property because this doc focuses on the Consumption logic app resource type, so customers wouldn't select the **Standard** option and thus view the **Publish** property. If you want this property documented, I've made the following edits. 

```suggestion
   | **Publish** | **Workflow** | Available only when you select the [**Standard** logic app resource type](create-single-tenant-workflows-azure-portal.md). By default, **Workflow** is selected for deployment to [single-tenant Azure Logic Apps](single-tenant-overview-compare.md) and creates an empty logic app resource where you add your first workflow. <p><p>**Note**: Currently, the **Docker Container** option requires a [*custom location*](../azure-arc/kubernetes/conceptual-custom-locations.md) on an Azure Arc enabled Kubernetes cluster, which you can use with [Azure Arc enabled Logic Apps (Standard)](azure-arc-enabled-logic-apps-overview.md). The resource locations for your logic app, custom location, and cluster must all be the same. |
```

Author: I get that but on the title it doesn't mention Consumption or Standard only create first Logic App

Reviewer: This doc is a quickstart for a beginner, so having the title say "Consumption" won't mean much to someone new to Azure Logic Apps. Still, the intro does mention that the quickstart creates a Consumption-based logic app resource.

   | **Region** | <*Azure-region*> | The Azure datacenter region where to store your app's information. This example uses "West US". <p>**Note**: If your subscription is associated with an [integration service environment](connect-virtual-network-vnet-isolated-environment-overview.md), this list includes those environments. |

>    | **Enable log analytics** | **No** | Only available for Consumption Type. <p><p>Change this option only when you want to enable diagnostic logging. For this example, leave this option unselected. |

Reviewer: ```suggestion
   | **Enable log analytics** | **No** | Available only when you select the **Consumption** logic app resource type. <p><p>Change this option only when you want to enable diagnostic logging. For this example, leave this option unselected. |
```

Author: I like the changes

Reviewer: Great! Please click the button that integrates the suggestions into your PR.

Reviewer: Hi @sandroasp, to move forward with the PR, you have to manually accept the suggestions I made. Thanks!

Reviewer 2: Hi @ecfan - I've accepted the changes you requested.
